### PR TITLE
Add See5 WebsiteBuilder

### PR DESCRIPTION
**Pull request by hosina**
Add See5 WebsiteBuilder

### CODE REVIEW

File: src/technologies/s.json [hunk 1]
```diff
@@ -7229,5 +7229,29 @@
...
+    "headers": {
+      "X-Powered-See5": "See5"
+    },  
+    "icon": "See5.svg",
+    "implies": [
+      "PHP","React","Laravel","Node.js","Django","Docker","Python","MongoDB"
+    ],
+    "saas": true,
+    "pricing": ["freemium", "recurring"],
+     "website": "https://see5.net"
```
Comment (nurbek91) [inline src/technologies/s.json:7255]: page does not open

File: src/technologies/s.json [hunk 2]
```diff
@@ -7229,5 +7229,29 @@
...
+      11,
+      88,
+      51
+    ],
+    "description": "See5 Is a Saas Service For Building Websites",  
+    "html": [    
+      "<link rel=[\"']stylesheet[\"'] [^>]+/see5pb/",
+      "style[^>]*>[^>]*see5",
+      "script[^>]*>[^>]*see5"
+    ],
```
Comment (nurbek91) [inline src/technologies/s.json:7245]: It is not recommended to use it due to resource consumption.

File: src/technologies/s.json [hunk 3]
```diff
@@ -7229,5 +7229,29 @@
...
     },
     "website": "https://styled-components.com"
+  },
+  "See5": {
+    "cats": [
+      1,
+      11,
+      88,
+      51
+    ],
```
Comment (nurbek91) [inline src/technologies/s.json:7239]: it is desirable to indicate 1 category maximum 2

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks!!!
Could you please provide links to examples.

**Comment by hosina** [comment]
website:see5.net 
example:sitetesttarahi006.see5.net

**Comment by nurbek91** [comment]
> see5.net

**Comment by nurbek91** [comment]
> website:see5.net example:sitetesttarahi006.see5.net
But this site is built using Wordpress and wpBakery
image

**Comment by hosina** [comment]
see5 website builder is saas service for build website with wordpress with pagebuilder 

example site: https://sitetesttarahi006.see5.net/

**Pull request closed without merging**